core/controllers/question_editor_test.py

```diff
@@ -69,8 +69,9 @@ def test_post_with_non_admin_or_topic_manager_email_disallows_access(self):
         self.login(self.NEW_USER_EMAIL)
         csrf_token = self.get_new_csrf_token()
         self.post_json(
-            '%s/%s' % (feconf.NEW_QUESTION_URL, self.skill_id),
-            {}, csrf_token=csrf_token, expected_status_int=401)
+            '%s' % (feconf.NEW_QUESTION_URL), {
```

Comment:
Nit, here and below, you don't need the '%s'. Just say self.post_json(feconf.NEW_QUESTION_URL, ...)

Reply:
Done